apps/opik-backend/src/main/resources/liquibase/db-app-analytics/migrations/000007_change_spans_usage_value_to_nullable.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+--changeset thiagohora:change_decimal_precision
+
+ALTER TABLE ${ANALYTICS_DB_DATABASE_NAME}.spans
+    MODIFY COLUMN usage Map(String, Nullable(Int32));
```

Comment:
We try as much as possible to avoid nulls because they have a performance hit in ClickHouse. @andrescrz, what would be better here? Using null, or making it as a string and using empty as null and enforcing number at the application layer.

Reply:
We discussed to not to follow this approach. This is an implementation detail which affects performance. We can perfectly fulfil the user requirement with other alternatives without changing the low level schema.